### DISCUSSION TITLE
Save functions

### DISCUSSION THREAD

**Discussion by ahmed22pm7** [NationalSecurityAgency/ghidra#5113] [Q&A]
Hello how can i save the functions folder as a text seprated ![Screenshot_20230320_042511_com.microsoft.rdc.androidx.jpg](https://user-images.githubusercontent.com/105469178/226236606-af236446-9788-4737-840d-471708e4dd74.jpg)

**Comment by ryanmkurtz** [comment]
* Click `Window -> Functions`
* Right click on table
* `Export -> Export To CSV...`
